helix-term/src/commands.rs

```diff
@@ -4361,7 +4361,7 @@ fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
             ("a", "Argument/parameter (tree-sitter)"),
             ("o", "Comment (tree-sitter)"),
             ("t", "Test (tree-sitter)"),
-            ("m", "Matching delimiter under cursor"),
+            ("m", "Closest pair to cursor"),
             (" ", "... or any character acting as a pair"),
```

Comment:
Oddly the "..." kinda confuses me. Along with the space on the left-hand-side it makes me think I can press any character and have that be matched. I think we could just remove the "..."?

Reply:
You can press any character and it will be matched! Even `b`. This is helpful when you have a regex in PHP that you've delimited with `#`, which is not a usual pair character, for instance.

Reply:
Ah, sweet! Let's keep this text then 👍